coremltools/converters/keras/_keras2_converter.py

```diff
@@ -239,6 +239,7 @@ def _convert(model,
     # (D) -> [D]
     # (Seq, D) -> [Seq, 1, D, 1, 1]
     # (Batch, Sequence, D) -> [D]
+    # (Batch, Seq, H, W, C) -> (C,H,W)
```

Comment:
If len(dim) == 4, then it would be a good idea to change the comment to (None, Seq, H, W, C) -> (C,H,W)